channels-rke2.yaml

```diff
@@ -630,6 +630,18 @@ releases:
         version: v3.23.3-build2022081001
     featureVersions: &featureVersions-v1
       <<: *featureVersions-v1-22-11-rke2r1
+  - version: v1.22.15-rc1+rke2r2
+    minChannelServerVersion: v2.6.3-alpha1
+    maxChannelServerVersion: v2.7.99
+    serverArgs: *serverArgs-v1-22-11-rke2r1
+    agentArgs: *agentArgs-v1-22-11-rke2r1
+    charts: &charts-v1-22-15-rke2r1
+      <<: *charts-v1
```

Comment:
Incomplete previous anchor to include from?

Reply:
Ah, lovely copy/pasta fun.